docs/en/connector-v2/source/Klaviyo.md

```diff
@@ -0,0 +1,162 @@
+# Klaviyo
+
+> Klaviyo source connector
+ 
+## Description
+
+Used to read data from Klaviyo.
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
+- [ ] [exactly-once](../../concept/connector-v2-features.md)
+- [x] [schema projection](../../concept/connector-v2-features.md)
+- [ ] [parallelism](../../concept/connector-v2-features.md)
+- [ ] [support user-defined split](../../concept/connector-v2-features.md)
+
+##  Options
+
+| name                        | type   | required | default value |
+| --------------------------- | ------ | -------- | ------------- |
+| url                         | String | Yes      | -             |
+| private_key                 | String | Yes      | -             |
+| revision                    | String | Yes      | -             |
+| method                      | String | No       | get           |
+| schema.fields               | Config | No       | -             |
```

Comment:
This field is usually defined as a `schema`

Reply:
Thanks for your review.I have changed it to be the same as http source.

![1668587369277](https://user-images.githubusercontent.com/45089228/202128334-e45a0003-f55f-4ee2-9b10-6ea36621cd8b.png)